gtsfm/configs/deep_front_end.yaml

```diff
@@ -0,0 +1,45 @@
+SceneOptimizer:
+  _target_: gtsfm.scene_optimizer.SceneOptimizer
+  save_gtsfm_data: True
+  save_two_view_correspondences_viz: False
+  save_3d_viz: True
+  pose_angular_error_thresh: 5 # degrees
+
+  feature_extractor:
+    _target_: gtsfm.feature_extractor.FeatureExtractor
+    detector_descriptor:
+      _target_: gtsfm.frontend.detector_descriptor.superpoint.SuperPointDetectorDescriptor
+
+
+  two_view_estimator:
+    _target_: gtsfm.two_view_estimator.TwoViewEstimator
+    corr_metric_dist_threshold: 3 # in px
+    matcher:
+      _target_: gtsfm.frontend.matcher.superglue_matcher.SuperGlueMatcher
+
+    verifier:
+      _target_: gtsfm.frontend.verifier.ransac.Ransac
+      use_intrinsics_in_verification: True
+    
+
+  multiview_optimizer:
```

Comment:
Is some form of inheritance supported in these YAML files? That way, we could have a default configuration in default.yaml, include it in deep_front_end.yaml and change/override only the feature_extractor and two_view_estimator.

Reply:
+1 to this.